### ISSUE TITLE
🐛 BUG: WebSocket Next.js HMR

### ISSUE THREAD

**Issue by DevinWinata** [cloudflare/wrangler2#1705]
### What version of `Wrangler` are you using?

2.0.26

### What operating system are you using?

Pop!_OS 22.04 LTS

### Describe the Bug

i think this might be a duplicate for WebSocket feature like #70 or #691, but for me it's a bit confusing because #1431 should add/fix this feature, right? or it still not being implemented? #1616.

here's the error, just in case it's a bug:
```
[pages:err] Error: Could not proxy request: TypeError: fetch failed
    at file:///home/devin/.local/share/pnpm/global/5/.pnpm/wrangler@2.0.26/node_modules/wrangler/miniflare-dist/index.mjs:4952:27
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at next (/home/devin/.local/share/pnpm/global/5/.pnpm/wrangler@2.0.26/node_modules/wrangler/templates/pages-template-worker.ts:149:22)
    at EventTarget.[kDispatchFetch] (/home/devin/.local/share/pnpm/global/5/.pnpm/@miniflare+core@2.6.0/node_modules/@miniflare/core/src/standards/event.ts:359:13)
    at /home/devin/.local/share/pnpm/global/5/.pnpm/@miniflare+http-server@2.6.0/node_modules/@miniflare/http-server/src/index.ts:290:20
    at Server.<anonymous> (/home/devin/.local/share/pnpm/global/5/.pnpm/@miniflare+http-server@2.6.0/node_modules/@miniflare/http-server/src/index.ts:398:24)
GET /_next/webpack-hmr 502 Bad Gateway (122.03ms)
```

other software version details:
node 16.17.0 with nvm 0.39.1
pnpm 7.9.3

**Comment by Skye-31** [comment]
This will be solved with #1616, like you said 🙂

**Comment by DevinWinata** [comment]
thanks for clarifying 😀
